### PR TITLE
Fix NFS volume mounting; the NFS was recreated but still using the old IP

### PR DESCRIPTION
**Pull request by jlewi**
* Add a playbook for dealing with  the NFS volume being full.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/testing/347)

### CODE REVIEW

File: playbook.md [hunk 1]
```diff
@@ -49,4 +49,103 @@ This is a playbook for build cops to help deal with problems with the CI infrast
...
+     labels:
+     ...
+     ```
+
+     * Make sure you have the field finalizers and its an empty list
+
+   * Update the object
+
+     ```
+     kubectl apply -f /tmp/nfs-external.yaml
```
Comment (richardsliu) [inline playbook.md]: Should we also check in this yaml somewhere?

Comment (jlewi) [reply playbook.md]: It is checked in. 
https://github.com/kubeflow/testing/blob/master/test-infra/ks_app/components/nfs-external.jsonnet

That's the original spec. The spec is then modified by the controller and the finalizers are added. So we are doing a read-modify-write operation here.

**Pull request merged**